challenges/2022-02-14-three-number-sort/ThreeNumberSort.cs

```diff
@@ -0,0 +1,76 @@
+using Xunit;
+using System.Collections.Generic;
+using System.Linq;
+
+public class ThreeNumberSort
+{
+  // I DON"T KNOW HOW TO INITIALIZE THIS LINE, HELP PLEASE!
+  public int SortArray()
```

Comment:
I need help with this line.

Reply:
I messed up and had to do this as a separate pull request, but my changes should make it work